utils/copy-env-vars/.eslintrc.js

```diff
@@ -0,0 +1,58 @@
+const baseParserOptions = {
```

Comment:
Can this PR wait until my PR has merged as it will override few `.eslintrc.js` and `tsconfig` changes.